.changeset/old-tables-own.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit/reactive-element': patch
+---
+
+Bind this to custom attribute converter methods
```

Comment:
We'll want to bump `lit` along with `reactive-element` and backticks around `this` so it's more apparent that it's referring to the keyword.
```suggestion
---
'@lit/reactive-element': patch
'lit': patch
---

Bind `this` to custom attribute converter methods
```